### PR TITLE
Add snap election section

### PR DESCRIPTION
**Pull request by wipfli**
Adds a section about how Voting Members can call a snap election (election which takes place earlier than the usual one year period) in case they don't trust the Governing Board anymore.

### CODE REVIEW

**Comment by lseelenbinder** [comment]
Overall looks good, but some of the language is vague, and, if the board is in a position of non-confidence, not defining “major decision” gives the board immense latitude to do damage while adhering to the charter.

What if we say “may not authorize new expenditures or make any changes to personnel,” etc.?

**Comment by wipfli** [comment]
Good idea. Can you propose changes?

File: CHARTER.md [hunk 1]
```diff
@@ -100,5 +100,10 @@ Governing Board members receive no remuneration by the MapLibre Organization for
 ### Replacement of Governing Board Members
 In the event of a Governing Board member resigning from the Board during a term, the nominees from the last election with the largest number of votes shall be contacted with a proposal to serve in the Government Board until the next election is called.
 
+### Snap Election
+
+A snap election of the Governing Board, i.e. an election which takes place earlier than the usual one year period, can be called if 50 percent of Voting Members who participated in the last election sign a no-confidence motion.
+
+If a no-confidence motion passes, the current Governing Board shall not take any major decisions until the election. Within two weeks, people can register themselves for the election with a self-nomination. Two weeks after the self-nomination deadline, the snap election shall take place, i.e. four weeks after a no-confidence motion passes.
```
Comment (wipfli) [inline CHARTER.md:107]:
```suggestion
If a no-confidence motion passes, the current Governing Board shall not take any major decisions until the election. In that time the Governing Board shall:

* not authorize new expenditures,
* not make any changes to personnel.

Within two weeks, people can register themselves for the election with a self-nomination. Two weeks after the self-nomination deadline, the snap election shall take place, i.e. four weeks after a no-confidence motion passes.
```

File: CHARTER.md [hunk 2]
```diff
@@ -102,6 +102,17 @@ Governing Board members receive no remuneration by the MapLibre Organization for
 ### Replacement of Governing Board Members
 In the event of a Governing Board member resigning from the Board during a term, the nominees from the last election with the largest number of votes shall be contacted with a proposal to serve in the Government Board until the next election is called.
 
+### Snap Election
+
+A snap election of the Governing Board, i.e. an election which takes place earlier than the usual one year period, can be called if 50 percent of Voting Members who participated in the last election sign a no-confidence motion.
+
+If a no-confidence motion passes, the current Governing Board shall not take any major decisions until the election. In that time the Governing Board shall:
```
Comment (HarelM) [inline CHARTER.md:109]: Is there a definition for "major decisions"?
Otherwise this looks good, thanks!

Comment (wipfli) [inline CHARTER.md:109]:
```suggestion
If a no-confidence motion passes, the current Governing Board shall until the next election not take any major decision, such as but not limited to:
```

File: CHARTER.md [hunk 3]
```diff
@@ -102,6 +102,17 @@ Governing Board members receive no remuneration by the MapLibre Organization for
...
 In the event of a Governing Board member resigning from the Board during a term, the nominees from the last election with the largest number of votes shall be contacted with a proposal to serve in the Government Board until the next election is called.
 
+### Snap Election
+
+A snap election of the Governing Board, i.e. an election which takes place earlier than the usual one year period, can be called if 50 percent of Voting Members who participated in the last election sign a no-confidence motion.
+
+If a no-confidence motion passes, the current Governing Board shall until the next election not take any major decision, such as but not limited to:
+
+* not authorize new expenditures,
+* not make any changes to personnel.
```
Comment (wipfli) [inline CHARTER.md:112]:
```suggestion
* authorize new expenditures,
* make any changes to personnel.
```

**Comment by wipfli** [comment]
Thanks Harel, the major decisions are now outlined by two examples (spending and personnel).

**Review by birkskyum — APPROVED** [review]
In its newest revision, I think it's explicit enough to enter the charter.

**Comment by wipfli** [comment]
Thanks for the reviews! I will merge this now and then we can still discuss and modify it in #74...

**Pull request merged**